plugins/module_utils/vmware.py

```diff
@@ -136,8 +136,11 @@ def find_object_by_name(content, name, obj_type, folder=None, recurse=True):
 
     objects = get_all_objs(content, obj_type, folder=folder, recurse=recurse)
     for obj in objects:
-        if unquote(obj.name) == name:
-            return obj
+        try:
```

Comment:
Could you please add a changelog entry for this change? Thanks.

Reply:
I've updated the entry for #791